### ISSUE TITLE
can I use cms as Yii2 module?

### ISSUE THREAD

**Issue by valetzyzy** [noumo/easyii#120]
I.E
In config/web.php
`'modules' => [
        'cms' => [
            'class' => 'dektrium\user\Module'
],`
and use cms as independent module

**Comment by boehsermoe** [comment]
Yes. Exactly that is what happen in https://github.com/noumo/easyii/blob/master/config/easyii.php

In the easyii-start this config will be loaded [here](https://github.com/noumo/easyii-start/blob/master/app/config/web.php)
